### DISCUSSION TITLE
Motherboard Integration?

### DISCUSSION THREAD

**Discussion by shaystings** [devcompl/TTController#110] [Idea]
I understand it's probably years out. But is that something on the roadmap?

**Comment by MoshiMoshi0** [comment]
What do you mean by motherboard integration?

**Reply by shaystings** [reply]
Some way to sync motherboard rgb

**Reply by MoshiMoshi0** [reply]
Then no that is not on the roadmap.
This software is only for themaltake controllers, only options is to write a bridge software that reads colors from ttcontroller and sends them to another software.
